Tests/MacroToolkitTests/MacroToolkitTests.swift

```diff
@@ -428,6 +445,13 @@ final class MacroToolkitTests: XCTestCase {
 
         // Pretty cursed
         XCTAssert(NilLiteral(nilLiteral)?.value != nil)
+
+        assertInlineSnapshot(of: NilLiteral(nilLiteral), as: .description) {
```

Comment:
Same as for RegexLiteral, this should really be testing `.value` not the initializer (this probably doesn't benefit from being a snapshot test).

Probably only include the snapshot or the XCTAssert, not both.